grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -675,22 +675,18 @@ class StockMovementService {
                 'max': max ? max.toInteger() : null,
         ]);
 
-        def editItemsIds = data.collect { it.id }
```

Comment:
I think you can fix this by changing it to:
`def editItemsIds = data.collect { "'$it.id'" }.join(',')`
so the editItemsIds is a string and looks like this: "'id1','id2','id3','id4'"

Reply:
Unfortunately it's still not working after adding it.